src/general/MainMenu.tscn

```diff
@@ -504,6 +550,100 @@ rect_min_size = Vector2( 400, 350 )
 WindowTitle = "MOD_LOAD_ERRORS"
 ErrorMessage = "MOD_LOAD_ERRORS_OCCURRED"
 
+[node name="SocialMedia" type="Control" parent="."]
```

Comment:
Instead of using plain Control to group the icons, use HBoxContainer to arrange them horizontally automatically. Then, you need to set its anchors to the bottom left so that it stays there on different screen sizes.